build.gradle

```diff
@@ -0,0 +1,30 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+
+group = 'net.sf.kxml'
+version = '2.3.1'
```

Comment:
Should be the latest tag (+1?)

Reply:
I am still figuring out how to update an existing pull request :-)
Can you please update it?

Reply:
Committ to your branch and do a push, and the pull request will get updated automatically.